code/game/objects/effects/landmarks.dm

```diff
@@ -130,8 +130,8 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 	icon_state = "Chief Engineer"
 
 /obj/effect/landmark/start/head_of_personnel
-	name = "Head of Personnel"
-	icon_state = "Head of Personnel"
+	name = "Executive Officer"
```

Comment:
As this is a core file, it needs to contain an //NSV13 tag

Reply:
Pretty sure that this is not an actual issue that can be noticed in game. It's probable that all our maps do this.

Reply:
No, that is a rebasing issue.